python/private/config_settings.bzl

```diff
@@ -161,74 +118,65 @@ def is_python_config_setting(name, *, python_version, reuse_conditions = None, *
         visibility = kwargs.get("visibility", []),
     )
 
-def construct_config_settings(name = None):  # buildifier: disable=function-docstring
+def alias_config_settings(name = None, values = VERSION_FLAG_VALUES.keys()):  # buildifier: disable=function-docstring
+    """alias the config settings to the current package.
+
+    Args:
+        name: A dummy value
+        values: {type}`list[str]` The list of version values.
+    """
+    for name in [
+        "python_version",
+    ] + [
+        "is_python_version_unset" if not version else ("is_python_" + version)
+        for version in [""] + values
+    ]:
+        native.alias(
```

Comment:
These aliases seem gratuitous. How about just loading a list from pythons_hub instead?

However, see main comment for another idea that might just make all the config_setting generation much simpler.